builtin/v13/migration/miner.go

```diff
@@ -0,0 +1,364 @@
+package migration
```

Comment:
This file is specific to the nv21 migration. I would propose dropping the entire `migration` directory for now -- we'll have to add the "lightweight" migration, but we can land this PR without that.